Gopkg.toml

```diff
@@ -0,0 +1,3 @@
+[[constraint]]
+  name = "github.com/inconshreveable/mousetrap"
```

Comment:
What we are using this `mousetrap` for? I don't see anything related this in the codebase.

Reply:
@praveenkumar It is for cross compiling Windows binary

Reply:
Wish we can have a mock in-place and disable it for now (as it is a single function with an interface):

https://github.com/spf13/cobra/blob/1e58aa3361fd650121dceeedc399e7189c05674a/cobra.go#L50-L54

It is not something we will use...

Reply:
@gbraad I agree, this is something unwanted dep for us with no use atm, we can add it back or put a mock later.

Reply:
@gbraad @praveenkumar Sorry confused about your comments. Are you suggesting that  this is fine for now but later we will remove the dependency and mock it in our code.

Reply:
I am fine either way... I do not believe this library adds value.
so if we mock it away, great... if you add it now, also OK!